libraries/shared/streamer/statediff_streamer.go

```diff
@@ -15,21 +15,20 @@
 package streamer
 
 import (
-	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/ethereum/go-ethereum/statediff"
 	"github.com/makerdao/vulcanizedb/pkg/core"
 	"github.com/sirupsen/logrus"
 )
 
 type Streamer interface {
-	Stream(chan statediff.Payload) (*rpc.ClientSubscription, error)
+	Stream(chan statediff.Payload) (core.Subscription, error)
```

Comment:
on the note of shuffling, I was pretty tempted to move the `Streamer` to something under `pkg` (probably `core` for interface and `eth` for implementation), but decided to hold off on that refactor so as not to overly expand the diff